streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTestDriver.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.test;
+package org.apache.kafka.streams.processor.internals;
```

Comment:
I added a package local method ProcessorContextImpl.getStateMgr() for a test to use. The state manager is our internal object so I don't want the method to be public. For a test code to use this method it has to be in the same package.